_episodes/37.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: episode
+title:  "35: Packaging and modernizing Trino"
+date: 2022-04-21
+tags: tarball rpm docker container java 17
+youtube_id: ""
+wistia_id: ""
+sections:
+   - title: "Releases 382 to 385"
+     desc: "Details about features and fixes in the new releases"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "tbd"
+     time: 0
+   - title: "Pull requests of the episode"
+     desc: "PR 12259: Support updating Iceberg table partitioning"
+     time: 0
+   - title: "Demo of the episode"
+     desc: "Iceberg Table Partition migrations"
+     time: 0
+   - title: "Question of the episode"
+     desc: "Can I force a pushdown join into a connected data source?"
+     time: 0
+---
+
+## Releases 382 to 385
+
+Official highlights from Martin Traverso:
+
+[Trino 382](https://trino.io/docs/current/release/release-382.html)
+
+* 
+
+Additional highlights worth a mention according to Manfred:
+
+* 
+
+More detailed information is available in the release notes for
+[Trino 382](https://trino.io/docs/current/release/release-382.html),
+[Trino 383](https://trino.io/docs/current/release/release-383.html),
+[Trino 384](https://trino.io/docs/current/release/release-384.html),
+and
+[Trino 385](https://trino.io/docs/current/release/release-385.html).
+
+## Concept of the episode: 
+
+## Pull requests of the episode: PR 12259: Support updating Iceberg table partitioning
+
+https://github.com/trinodb/trino/pull/12259
+
+## Demo of the episode: Iceberg Table Partition migrations
+
+
+
+## Question of the episode: Can I force a pushdown join into a connected data source?
```

Comment:
I think that you can discuss the join pushdown we have: conditions, limitations, etc. Join pushdown is a different story than query pass-through via PTF, especially that it does not require modifying the query in any way to work. 

After you explain that, you can mention query pass-through via PTF, which can be considered the alternative way to pushdown join, and mention that it will be covered in another episode.